src/Signup.js

```diff
@@ -27,37 +27,37 @@ export default function Signup(props) {
     const { name, value } = event.target;
     if (name === "password")
       setPerson({
-        firstname: person["firstname"],
```

Comment:
Should these be single quotes for consistency?

Reply:
honestly I think it is fine for the frontend to have a slight different style than backend. 

my formatter default behavior is to use double quotes. I think it just has to be consistent. 

I am planning to run my formatter through all the files before we send it in for grading. 

@anthony-yeo

Reply:
yeah, I fell into using the double quotes to fit in with what was already being used in the frontend - but if Ash is formatting all the files then I think we are good.